src/cuttlefish_conf.erl

```diff
@@ -122,6 +123,11 @@ generate_element(MappingRecord) ->
             lager:warning("{level, ~p} has been deprecated. Use 'hidden' or '{hidden, true}'", [Level])
     end,
 
+    Default = case ConfFileDefault of
```

Comment:
Feels to me like `config_file_default/1` should do this logic and return "the right thing" on its own.